cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+          } else if ( key === 9 && !event.shiftKey ) {
+            if ( self.fieldset.getAttribute( 'visibility' ) === 'visible' ) {
+              self.close();
+            }
+          }
+        }
+      } );
+
+      $.bind( $( this.list ), {
+        mousedown: function( event ) {
+          self.skipBlur = true;
+        },
+        keydown: function( event ) {
+          var key = event.keyCode;
+          var checked = event.target.checked;
+          console.log( key );
+
+          if ( key === 13 ) {
+            event.preventDefault();
+            if ( checked ) {
+              event.target.checked = false;
+            } else {
+              event.target.checked = true;
+            }
+            $.fire( $( event.target ), 'change', {} );
+          } else if ( key === 27 ) {
+            self.search.focus();
+            self.close();
+          } else if ( key === 38 ) {
+            event.preventDefault();
+            self.previous();
+          } else if ( key === 40 ) {
+            event.preventDefault();
+            self.next();
+          }
+        }
+      } );
+
+      forEach( inputs, function( i ) {
+        $.bind( $( inputs[i] ), {
+          change: function( event ) {
+            self.updateSelections( event.target.id );
+            self.resetSearch();
+          }
+        } );
+      } );
+    }
+  };
+
+  // Static methods/properties
+  _.all = [];
+
+  _.FILTER_CONTAINS = function ( text, input ) {
+    return RegExp( $.regExpEscape( input.trim() ), 'i' ).test( text );
+  };
+
+  // Private Functions
+  function configure( properties, options ) {
+    for ( var i in properties ) {
+      var initial = properties[i];
+      var attrValue = this.elem.getAttribute( 'data-' + i.toLowerCase() );
+
+      if ( typeof initial === 'number' ) {
+        this[i] = parseInt( attrValue );
+      } else if ( initial === false ) { // Boolean options must be false by default anyway
+        this[i] = attrValue !== null;
+      } else if ( initial instanceof Function ) {
+        this[i] = null;
+      } else {
+        this[i] = attrValue;
+      }
+
+      if ( !this[i] && this[i] !== 0 ) {
+        this[i] = ( i in options ) ? options[i] : initial;
+      }
+    }
+  }
+
+  // Helpers
+  var slice = Array.prototype.slice;
+
+  function hasClass( element, className ) {
+    return !!element.className.match( new RegExp( '(\\s|^)' + className + '(\\s|$)' ) );
+  }
+
+  function forEach( array, callback, scope ) {
+    for ( var i = 0; i < array.length; i++ ) {
+      callback.call( scope, i, array[i] );
+    }
+  }
+
+  function valInArray( array, val ) {
+    var match = false;
+    if ( !array.length > 0 ) {
+      return match;
+    } else {
+      array.forEach( function( item, index ) {
+        if ( item.value === val ) {
+          match = {
+            index: index,
+            item: item
+          };
+        }
+      } );
+    }
+
+    return match;
+  }
+
+  function $( expr, con ) {
```

Comment:
Since this is a fork of awesomeplete, I tried to stick to the original formatting and organization as closely as possible. After being away for a bit, I'm less attached to this being a fork and rather just a derivative of that project.